MoneySplitter.Services/Executor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using MoneySplitter.Infrastructure;
+using MoneySplitter.Models;
+using System;
+using System.Threading.Tasks;
+
+namespace MoneySplitter.Services
+{
+    public class Executor : IExecutor
+    {
+        public async Task ExecuteWithRetryAsync(Func<Task> asyncAction)
+        {
+            for (int i = 0; i < Defines.Executor.COUNT_TRY; i++)
+            {
+                await ExecuteOneTime(asyncAction);
```

Comment:
As I see you executing all actions N times?

Reply:
fixed